src/python/bcc/table.py

```diff
@@ -186,6 +186,10 @@ def clear(self):
         for k in self.keys():
             self.__delitem__(k)
 
+    def zero(self):
+        for k in self.keys():
+            self[k] = self.Leaf()
```

Comment:
yeah found it. so it will call update_elem() which will do kmalloc in user context which is ok.